modules/images-configuration-registry-mirror.adoc

```diff
@@ -51,8 +66,31 @@ Each of these custom resource objects identify the following information:
 requested from the source repository.
 --
 
+ifndef::winc[]
 For new clusters, you can use IDMS, ITMS, and ICSP CRs objects as desired. However, using IDMS and ITMS is recommended.
 
 If you upgraded a cluster, any existing ICSP objects remain stable, and both IDMS and ICSP objects are supported. Workloads using ICSP objects continue to function as expected. However, if you want to take advantage of the fallback policies introduced in the IDMS CRs, you can migrate current workloads to IDMS objects by using the `oc adm migrate icsp` command as shown in the *Converting ImageContentSourcePolicy (ICSP) files for image registry repository mirroring* section that follows. Migrating to IDMS objects does not require a cluster reboot.
 
-include::snippets/idms-global-pull-secret.adoc[]
+[NOTE]
+====
+If your cluster uses an `ImageDigestMirrorSet`, `ImageTagMirrorSet`, or `ImageContentSourcePolicy` object to configure repository mirroring, you can use only global pull secrets for mirrored registries. You cannot add a pull secret to a project. So there
+====
+endif::winc[]
+
+ifdef::winc[]
+[NOTE]
+====
+If your cluster uses an `ImageDigestMirrorSet` or `ImageTagMirrorSet` object to configure repository mirroring, you can use only global pull secrets for mirrored registries.
```

Comment:
Do we need to mention that we allow pulling an image directly from a disconnected registry, deployment config requires adding pull secret on that workloads namespace

Reply:
@rrasouli @saifshaikh48 I made [some changes based on Ronnie's comment](https://github.com/openshift/openshift-docs/pull/74918/commits/c33490bf3aa45fcf9c81db105508468d09cbbfea#diff-038a6106c09d0b2f6ac1faf140b33c1091d367447b46c12fd8384da289646369). I changed the focus from using WMCO in a  disconnected cluster to using WMCO with a mirror registry. And added that you can pull images from the mirror registry directly without needed a IDMS/ITMS.

Reply:
I think it goes without saying that images in disconnected registries can be referenced directly. That is the case regardless of whether the cluster is in a connected or disconnected network